model.py

```diff
@@ -312,9 +312,11 @@ class SessionManager(object):
 
     MATERIALIZED_VIEW_WORKS = 'mv_works_editions_datasources_identifiers'
     MATERIALIZED_VIEW_WORKS_WORKGENRES = 'mv_works_editions_workgenres_datasources_identifiers'
+    MATERIALIZED_VIEW_LANES = 'mv_works_for_lanes'
     MATERIALIZED_VIEWS = {
         MATERIALIZED_VIEW_WORKS : 'materialized_view_works.sql',
```

Comment:
You can remove this file and the SessionManager references MaterializedWork. (Unless it's for rollback purposes?)

Reply:
It's in case we try it this way and we need to go back, but it's also so we can test on a running site with a big dataset, without modifying the database tables used by the production code.